src/plugins/intel_cpu/tests/functional/single_layer_tests/shape_ops.cpp

```diff
@@ -280,6 +281,53 @@ const auto params_EmptyTensor = ::testing::Combine(::testing::Values(shape_Empty
 
 INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_EmptyTensor, ShapeOpsCPUTest, params_EmptyTensor, ShapeOpsCPUTest::getTestCaseName);
 
+
+TEST(smoke_reshape, need_shape_infer) {
+    ov::ParameterVector inputs{std::make_shared<ov::op::v0::Parameter>(ov::element::i32, ov::PartialShape{-1, -1}),
+                               std::make_shared<ov::op::v0::Parameter>(ov::element::i32, ov::PartialShape{2})};
+    auto dataInput = inputs.front();
+    auto secondaryInput = inputs.back();
+    auto shapeOps = std::make_shared<ngraph::opset1::Reshape>(dataInput, secondaryInput, true);
+    CPUTestsBase testBase;
+    auto function = testBase.makeNgraphFunction(ov::element::i32, inputs, shapeOps, "ReshapNeedShapeInfer");
```

Comment:
Why didn't you simply instance yet another parametrized test suite for the existing test class? It looks more straightforward then writing a separate unit test. All you need is to write a specific sequence of the input shapes. Comparison with the reference also implies comparison of the shapes.

Reply:
thanks your suggestion, fixed it